src/components/navbar.js

```diff
@@ -1,8 +1,34 @@
 /** @jsx jsx */
 import { useCallback, useState } from 'react'
-import { Box, Text, jsx } from 'theme-ui'
+import { Box, Image, Flex, Text, jsx } from 'theme-ui'
 import useAuth from '../util/auth'
-import Link from './link'
+import logo from '../../static/climatedu.png'
+import NavButton from './navbutton'
+import { BsPerson } from 'react-icons/bs'
+
+const links = [
+  {
+    'location':_=>'/',
+    'text':_=>'Home'
+  },
+  {
+    'location':_=>'/about',
+    'text':_=>'About',
+    'dropdown':_=>[
+      {'location':'/about/team','text':'Our Team'},
+      {'location':'/about/mission','text':'Our Mission'}
+    ]
+  },
+  {
+    'location':_=>'/contact',
+    'text':_=>'Contact Us'
+  },
+  {
+    'location':user=>user?'/profile':'/login',
+    'text':user=>user?'Welcome '+user.displayName+'!':'Log in',
+    'icon':_=>BsPerson
+  },
+];
```

Comment:
Having each item as a function is bad practice, instead have a flag indicating whether it should be shown when logged in or out; the list should be pure data, without computation.